Robot_Adapter/Read/Read.cs

```diff
@@ -93,7 +93,7 @@ protected override IEnumerable<IBHoMObject> Read(Type type, IList indices = null
                 return ReadLoads(); //TODO: Implement load extraction
 
             if (type.IsGenericType && type.Name == typeof(BHoMGroup<IBHoMObject>).Name)
```

Comment:
Could change this to `            if (typeof(IBHoMGroup).IsAssignableFrom(type))` to allow using the interface as well, but not critical. Will approve, if you have time to look at this today @rwemay feel free to update, if not, I will merge first thing tomorrow morning.